sofiadinizms/NFT-onepage
# Feature/references section

## What I did
1. Created the new References sections, made it responsive and connected it to the backend

## How to test

1. Run the project (yarn on the front, npm on the back)
2. Go to your local host 3001
3. Compare the references section with the new mockup
4. Check if it's responsive
5. Try to change the db info and see if the front-end reflects those changes

## Technical debts
1. Errors on the console
